reliability.md

```diff
@@ -1,33 +1,45 @@
+
+A second way is manually, through the operating system, so that you mount a system that makes a copy of all files (for example in a ZIP) and on the other hand to dump the database to obtain a copy of the SQL file with all the data.
+
+Finally, and most often (in addition to what the provider can provide) is to have a backup plugin. There are many plugins and in general we recommend that you try several until you find the one that suits your needs. It is not the same to make a backup of a small site (200 MB) as to make a large one (5 GB). The response times and performance of the server could cause the system to become saturated and not perform the copies, so we recommend that you try the different systems until you find the one that works best.
+
+It's also important that when you back up your data, it's stored in at least two different locations. One is usually the server where the website is located. Sometimes the site gets corrupted but it doesn't affect the machine, so having the files close by and available quickly is usually very helpful. The other case is to have them outside the server, in an external place, and even outside the same network or provider or country. The fact that it is in another place would allow that even if that provider had a serious problem (and for example you had no possibility to access it) you could restore your site from an external place.
 
-Site monitoring systems and services can notify you when your site isn't working properly. They can often correct any minor issues, or help you to do so before they become major issues.
+### When to make copies
 
-### Uptime Monitoring
+You can make as many copies as you want, though in moderation, like everything else. In general, it will depend on how many changes you make to your website and how often to determine how much data you are willing to lose in an extreme case.
 
-Uptime monitoring is traditionally done at the server level or by checking one or more URLs on the site at regular intervals to make sure they are responding properly. A combination of internal and external uptime monitoring is ideal for users, and there exist a variety of software and services to handle this for you.
+For example, if you have a site where you publish content every week, and your visitors post some comments, a daily backup is probably enough.
+
+On the other hand, if you are a media outlet that publishes a hundred articles a day and there is a lot of interaction, you will probably want to make a copy every few hours.
+
+In any case, it is advisable to have at least one daily copy for a week, and then store a weekly copy for at least a few months. This way you'll be sure to have copies that go back 3-6 months. Again, this will depend on the size and configuration of your site.
```

Comment:
This section seems very user-centric and not host-centric. Is there a way to add recommendations on frequency from the host point of view?